docs/GETTING_STARTED.md

```diff
@@ -1,288 +1,242 @@
-ctr i push --platform linux/amd64 --plain-http localhost:5000/rabbitmq:latest
-```
+Instead of converting the image format, soci-snapshotter uses the SOCI index
+associated with an image to implement its lazy loading. For more detail
+please see [README](../README.md#no-image-conversion).
 
-**NOTE**: the platform tag might be different depending what environment you
-are working in.
+### Create SOCI index
 
-Once again let's confirm via a curl command:
-```
-curl http://localhost:5000/v2/_catalog
-```
-We should now see that our repository has been created
-`{"repositories":["rabbitmq"]}`.  We can further inspect and see what tags were
-created.
-```
-curl http://localhost:5000/v2/rabbitmq/tags/list
-```
-We can also look at the details of the tag using:
-```
-curl http://localhost:5000/v2/rabbitmq/manifests/latest
+Let's create a SOCI index, which later will be pushed to your registry:
+
+> The `--legacy-registry` flag specifies to create SOCI index as image manifest
+> instead of artifact.
+
+```shell
+sudo soci create --legacy-registry $REGISTRY/rabbitmq:latest
+
+# output
+layer sha256:57315aaee690b22265ebb83b5443587443398a7cd99dd2a43985c28868d34053 -> ztoc skipped
+layer sha256:ed46dea0429646ca97e7a90d273159154ab8c28e631f2582d32713e584d98ace -> ztoc skipped
+layer sha256:3f0e404c1d688448c1c3947d91d6e0926c67212f4d647369518077513ebdfd91 -> ztoc skipped
+layer sha256:626e07084b41a102f8bcedf05172676423d1c37b8391be76eee2d7bbf56ec31e -> ztoc skipped
+layer sha256:b49348aba7cfd44d33b07730fd8d3b44ac97d16a268f2d74f7bfb78c4c9d1ff7 -> ztoc skipped
+layer sha256:ec66df5c883fd24406c6ef53864970f628b51216e8e1f3f5981c439ed6e4ed41 -> ztoc skipped
+layer sha256:8147f1b064ec70039aad0068f71b316b42cf515d2ba87e6668cb66de4f042f5a -> ztoc skipped
+layer sha256:f63218e95551afe34f3107b1769a556a3c9a39279cb66979914215e03f4e2754 -> ztoc sha256:ccae6b7217b73ae9caf80bff4c5411dada341739c8b443791fba227b226c61d0
+layer sha256:7608715873ec5c02d370e963aa9b19a149023ce218887221d93fe671b3abbf58 -> ztoc sha256:740374aa7cac1764593430843d428a73a30d4a6a0d45fb171c369f3914a638eb
+layer sha256:96fb4c28b2c1fc1528bf053e2938d5173990eb12097d51f66c2bb3d01a2c9a39 -> ztoc sha256:dc9a2ca27d2b680279fc8052228772b9c03a779d0b7cc61012d2ad833ad1ff5e
+...
 ```
-## Create and push a SOCI index
-Now that we have confidence that our registry contains the image let's work on
-the SOCI index.  This is the pre-built index that allows random access of the
-compressed layer. For more detail please see [our
-README](https://github.com/awslabs/soci-snapshotter#no-image-conversion).
 
-First off, we need to build the SOCI project using the `make` command.  This
-will generate two binaries in the `out` directory.  You should see:
-- soci (this is the CLI tool)
-- soci-snapshotter-grpc (this is the daemon used for lazy loading, we will use
-this later)
+Behind the scene SOCI created two kinds of objects. One is a series of ztocs
+(one per layer). A ztoc is a table of contents for compressed data. The other is
```

Comment:
We should also briefly mention that layers that do not meet the minimum size threshold do not have ztocs generated for them for clarity.

Reply:
fixed